src/listener/handlers/document-update-handler.ts

```diff
@@ -152,32 +150,35 @@ class DocumentUpdateHandler implements IHandler {
     checkEligibleForFileValidation(contentValidationReport: ValidationReport, documentUrl: string, documentType: string): Promise<ValidationReport> {
         if (documentType.toUpperCase() === 'FILE') {
             // refresh doc before checking cloudUrl
-            return this.ingestClient.retrieveMetadataDocument(documentUrl).then((doc: any) => {
-                if (doc['cloudUrl']) {
-                    // if cloud url present, proceeds with file validation
-                    if (contentValidationReport.validationState.toUpperCase() == "VALID") {
-                        return this.attemptFileValidation(contentValidationReport, doc, documentType)
+            return this.ingestClient.retrieveMetadataDocument(documentUrl)
+                .then((doc: any) => {
+                    if (doc['cloudUrl']) {
```

Comment:
You could flatten these if/else statements a bit. I don't think you need the else at all if you just return early from the if